v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/reservations_client.py

```diff
@@ -502,9 +502,8 @@ async def __process_v2g_events(self, new_v2g_events):
 
     async def __set_events_in_main_app(self, v2g_args: str = ""):
         try:
-            await self.v2g_main_app.handle_calendar_change(
-                v2g_events=self.v2g_events, v2g_args=v2g_args
-            )
+            self.emit("calendar_change", v2g_events=self.v2g_events, v2g_args=v2g_args)
```

Comment:
This is the main change: instead of calling the method directly, it now uses an event.

Reply:
@ArdJonker I see this being called _before_ the event handler is created. Does that mean that the order of initialisation should be revisited?

Reply:
Yes indeed!

Reply:
Has been fixed